draft-ietf-quic-transport.md

```diff
@@ -2320,15 +2352,28 @@ Sequence:
   the value selected during the handshake comes immediately before the first
   value that a server can send.
 
+Length:
+
+: An 8-bit unsigned integer containing the length of the connection ID.  Values
+  less than 8 and greater than 22 are invalid and MUST be treated as a
+  connection error of type PROTOCOL_VIOLATION.
```

Comment:
You have the room here to spell it out, but there's something to be said for using the same encoding other places the length is written.  Maybe just use four bits and zero/ignore the upper four?

Reply:
There's a good reason to make the headers we expose to the cesspit use a crappy encoding, but less so here.